models/threshold/Threshold.adoc

```diff
@@ -0,0 +1,83 @@
+= ThresholdHealth
```

Comment:
I don't like to have a separated .adoc for this.

This should be defined at least in two places:
- Where all info related configuration is placed (typically in operator Kiali CR descriptor).
- Also, in Kiali.io where I guess this user-friendly style is very good (and also we would need to move the rest of config in that way).

Reply:
Ok I'll create another PR for Doc after discussion about what do do it with his